rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/monitoring/views/MonitoringSourceView.java

```diff
@@ -192,7 +192,11 @@ public void close() {
     @Override
     public void onFailure(Throwable caught) {
         close();
-        status.setContents(JSONUtils.getJsonErrorMessage(caught));
+        String errorMessage = JSONUtils.getJsonErrorMessage(caught);
+        if(errorMessage.contains("HTTP 500")){
+            errorMessage = "Monitoring is not available on this node source";
```

Comment:
You are right. In fact this change is not related to disabling monitoring. Sorry for mess. It was added to this pull request automatically when the commit was pushed. I noticed that usually we have "HTTP 500 Internal Server Error", when click on monitoring tab for Source Node. The aim of this commit to show proper message for this case.